src/nerves_key_pkcs11.c

```diff
@@ -176,9 +177,11 @@ CK_DEFINE_FUNCTION(CK_RV, C_GetSlotList)(
             if (i > max_count)
                 return CKR_BUFFER_TOO_SMALL;
 
-            if (pSlotList)
+            if (pSlotList) {
                 pSlotList[count] = i;
-            count++;
+                pSlotList[count+1] = i + 10;
+            }
+            count+=2;
```

Comment:
I can't suggest this change due to it crossing the lines in GitHub. However, it should be:

```
    if (count + 2 >= max_count)
        return CKR_BUFFER_TOO_SMALL;

    if (pSlotList) {
        pSlotList[count] = i;
        pSlotList[count + 1] = i + 10;
    }
    count += 2;
```

`count` being incremented outside of the `if` statement is correct. It's for calls that just want to know how many items would be added.